examples/virtual_sites/vsite_showcase.ipynb

```diff
@@ -10,17 +10,16 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    molecule = Molecule.from_smiles("c1cc(-Cl)ccc1C(=O)CS[C]1=CO[C](F)(F)CC1")
(not blocking) It's strange that this SMILES contains <code>(-Cl)</code> and <code>(F)</code> - Why is the single bond explicit in the first but not in the second?
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>